src/ui/server/controllers/insurance/business/company-details/save-and-back/index.POST-save-and-back.test.ts

```diff
@@ -79,26 +77,6 @@ describe('controllers/insurance/business/companies-details', () => {
 
         expect(res.redirect).toHaveBeenCalledWith(`${INSURANCE_ROOT}/${req.params.referenceNumber}${ALL_SECTIONS}`);
       });
-
-      it('should call mapAndSave.companyDetails once with the data from constructPayload function and company', async () => {
```

Comment:
is it not worth keeping these tests to double check as security related?

Reply:
Not sure why this was removed, have re-added a test for this.